packages/app-desktop/gui/NoteEditor/NoteBody/WhiteboardEditor/style.scss

```
@@ -0,0 +1,290 @@
+			letter-spacing: 0;
+			white-space: nowrap;
+			overflow: hidden;
+			text-overflow: ellipsis;
+		}
+	}
+
+	> .body {
+		flex: 1;
+		padding: 8px;
+		overflow: auto;
+		word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)